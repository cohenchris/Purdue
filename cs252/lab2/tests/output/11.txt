0x55d16248bf20
(nil)
[
	addr: 3872
	size: 192
	left_size: 3840
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
0x55d16248ce30
(nil)
[
	addr: 7728
	size: 432
	left_size: 3840
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
0x55d16248df84
0x55d16248ce30
[
	addr: 12164
	size: 192
	left_size: 3840
	allocated: false
	prev: NULLPTR
	next: 7728
]

[
	addr: 7728
	size: 432
	left_size: 3840
	allocated: false
	prev: 12164
	next: NULLPTR
]

