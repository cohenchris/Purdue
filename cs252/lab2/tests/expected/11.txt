
[
	addr: 3872
	size: 192
	left_size: 3840
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
[
	addr: 7728
	size: 432
	left_size: 3840
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
[
	addr: 12164
	size: 192
	left_size: 3840
	allocated: false
	prev: NULLPTR
	next: 7728
]

[
	addr: 7728
	size: 432
	left_size: 3840
	allocated: false
	prev: 12164
	next: NULLPTR
]
